File: Quarter-of-the-year/Quarter-of-the-year.py
def quarter_of(month):
  if type(month) == int :
    return (1 if month == 1 or month == 2 or month == 3 
        else 2 if month == 4 or month == 5 or month == 6
        else 3 if month == 7 or month == 8 or month == 9
        else 4 if month == 10 or month == 11 or month == 12
        else 'Not a valid month number')

File: Quarter-of-the-year/Quarter-of-the-year.js
const quarterOf = (month) => {
  if (typeof month === 'number') {
    return month === 1 || month === 2 || month === 3 ? 1
      : month === 4 || month === 5 || month === 6 ? 2
      : month === 7 || month === 8 || month === 9 ? 3
      : month === 10 || month === 11 || month === 12 ? 4
      : 'Not a valid month number';
  }
}